PyInstaller/utils/win32/winutils.py

```diff
@@ -28,12 +26,8 @@ def get_windows_dir():
     """
     Return the Windows directory e.g. C:\\Windows.
     """
-    try:
-        import win32api
-    except ImportError:
```

Comment:
Same thing here, if win32api isn't present then pyinstaller will exit, so no sense trying to detect this.